ui-v2/app/templates/components/discovery-chain.hbs

```diff
@@ -8,7 +8,7 @@
         border: var(--decor-border-100);
         border-radius: var(--decor-radius-300);
         border-color: var(--gray-500);
-        box-shadow: 0 8px 10px 0 rgba(0, 0, 0, 0.1);
+        box-shadow: (--decor-shadow-600);
```

Comment:
I think I tried using a CSS var here and I couldn't get it to work originally. Maybe you'll have better luck, but it will need the `var`.

This rule gives the selected state (when you click on a box) of the discovery-chain a shadow, so just double check that the shadow is applied.

Reply:
I forgot to update this one to the new variable "elevation" name, too. Thanks for catching that!

Reply:
Ah yeah 😂 good spot!